### PR TITLE
Developers Tools Page

### LINKED ISSUE #103: improve testnet faucet page
This form needs more information to be more user friendly.
https://app.telos.net/testnet/developers

The requirements will be specified later.

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #103 and #259

**Note:** All texts need to be approved by @pmjanus before putting them into all translated index files.

## Description
This PR improves the interaction with new developers by giving them a better welcome and separating the three available tools each in their own context. Also, each tab can be deeplinked from an external site using the parameter `tab`.

All the texts were added to the corresponding index following the i18n standard. Also, a lot of checks were introduced in the forms.

## Test scenarios
- Go to this [EVM tab on dev page](https://deploy-preview-335--telos-app-native-staging.netlify.app/testnet/developers?tab=tlos-evm)
  - You should be looking the EVM tab where you can send telos to any account on Testnet once a day
  - copy and paste a valid address and press the button send some telos
     - You should be notified if it succeeds.
  - Press the button again
     - You should get an error and be notified

## Screenshots
image

image

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file and/or created a new issue with the required translations for the currently supported languages
-   [x] I have tested for mobile functionality and responsiveness
-   [x] I have added appropriate test coverage

### CODE REVIEW

File: src/pages/testnet/DevelopersPage.vue [hunk 1]
```diff
@@ -12,137 +15,295 @@ export default {
...
-      )
+<template>
+    <q-page class="p-dev-page flex flex-center">
+        <div class="p-dev-page__title-card">
+            <div class="p-dev-page__title">Testnet Developers Tools</div>
+            <div class="text-subtitle2">
+                Use the following tools to create a new account,
+                send TLOS tokens to a Telos native account,
+                or send TLOS tokens to an EVM compatible address.
+            </div>
```
Comment (Viterbo) [inline src/pages/testnet/DevelopersPage.vue:105]: Julio, please help me write something intelligent here to welcome new developers.

File: src/pages/testnet/DevelopersPage.vue [hunk 2]
```diff
@@ -12,137 +15,295 @@ export default {
...
+            </q-tabs>
+    
+            <q-tab-panels class="p-dev-page__panels" v-model="tab" animated>
+                <q-tab-panel class="p-dev-page__panel" name="create">
+                    <q-card-section class="p-dev-page__panel-section">
+                        <div class="text-h6">Create a New Testnet Account</div>
+                        <div class="text-subtitle2">
+                            Here you can create a new account by providing a unique
+                            name and public keys for owner and active permissions.
+                        </div>
```
Comment (Viterbo) [inline src/pages/testnet/DevelopersPage.vue:121]: @pmjanus, also here: account creation

File: src/pages/testnet/DevelopersPage.vue [hunk 3]
```diff
@@ -12,137 +15,295 @@ export default {
...
+                        v-if="transactionId"
+                        class="p-dev-page__panel-btn p-dev-page__trx-id"
+                        color="secondary"><a href="#" targe="_blank">{{ transactionId }}</a></q-btn>
+                </q-tab-panel>
+        
+                <q-tab-panel class="p-dev-page__panel" name="tlos-native">
+                    <q-card-section class="p-dev-page__panel-section">
+                        <div class="text-h6">Send TLOS to Telos Account</div>
+                        <div class="text-subtitle2">Use this tab to send TLOS tokens to a Telos native account.</div>
+                    </q-card-section>
```
Comment (Viterbo) [inline src/pages/testnet/DevelopersPage.vue:166]: @pmjanus, also here: send TLOS to Native accounts

File: src/pages/testnet/DevelopersPage.vue [hunk 4]
```diff
@@ -12,137 +15,295 @@ export default {
...
+                        class="p-dev-page__panel-btn p-dev-page__trx-id"
+                        color="secondary"><a href="#" targe="_blank">{{ transactionId }}</a></q-btn>
+                </q-tab-panel>
+        
+                <q-tab-panel class="p-dev-page__panel" name="tlos-evm">
+                    <q-card-section class="p-dev-page__panel-section">
+                        <div class="text-h6">Send TLOS to EVM Address</div>
+                        <div class="text-subtitle2">
+                            Send TLOS tokens to an EVM compatible address with this option.
+                        </div>
```
Comment (Viterbo) [inline src/pages/testnet/DevelopersPage.vue:197]: @pmjanus, also here: send TLOS to EVM accounts

**Comment by donnyquixotic** [comment]
@Viterbo can you include input validation in this PR?

**Review by pmjanus — APPROVED** [review]
Text is approved

**Review by donnyquixotic — DISMISSED** [review]

**Pull request merged**